Rakefile

```
@@ -154,6 +154,19 @@ namespace :release do
     ShopifyCLI::Release.new(new_version, github_access_token).prepare!
     puts "Completed!"
   end
+
+  task :package do
+    github_access_token = ENV["GITHUB_ACCESS_TOKEN"]
```

What about taking the token from `dev` if there is no ENV var? We could better extract it to another method.

```suggestion
    github_access_token = ENV["GITHUB_ACCESS_TOKEN"] | `/opt/dev/bin/dev github print-auth | grep Password | awk '{print $NF}'`.chomp
```